chipy_org/settings.py

```diff
@@ -123,7 +127,7 @@ def env_list(key, defaults=None, delimiter=","):
 
 # Make this unique, and don't share it with anybody.
 SECRET_KEY = env_var("SECRET_KEY")
-
+DAY_TO_POST = int(env_var("DAY_TO_POST", 6))
```

Comment:
Hmmm, what is this for? I'd push to make any settings like this human-readable. DoW math is notoriously wrong with either Sun=0 or Sun=1.

Reply:
Leftover code will delete this.  Was going to do it as a env var, but moved it to a command argument.